gradle.properties

```diff
@@ -1,9 +1,9 @@
 version=9.2-SNAPSHOT
 org.gradle.jvmargs=-Xmx8G -XX:MaxMetaspaceSize=512M
 sonarlintCoreVersion=9.4.0.74778
-intellijBuildVersion=IC-2021.3
-clionBuildVersion=CL-2021.3
-riderBuildVersion=RD-2021.3
+intellijBuildVersion=IC-2022.3
+clionBuildVersion=CL-2022.3
+riderBuildVersion=RD-2022.2
```

Comment:
I think it's the first time we use a different version for one flavor. It means for Rider we build against a lower version than the one we officially support (as the real minimal version is defined in the `plugin.xml`). I don't know if this could be a problem, maybe if some APIs were dropped, but I don't think we used deprecated APIs. What is the reason for not using the same version?

Reply:
Rider 2022.3 has a bug that is well-known with Gradle. In order to support 2022.3 and forward I am using this version.